tests/unit/test-firefox/test.firefox.js

```diff
@@ -297,6 +297,46 @@ describe('firefox', () => {
 
   });
 
+  describe('useProfile', () => {
+
+    function withBaseProfile(callback) {
```

Comment:
@saintsebastian `withBaseProfile` this utility function is identical to the one already defined by the `copyProfile` tests (at line 172 of the same file).

We should move this helper into the head of this module, so that it can be shared by both `copyProfile` and `useProfile` without duplicating its definition.